example/lib/widgets/apply_method_item.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+
+class ApplyMethodItem extends StatefulWidget {
+  final String name;
+  final num defaultValue;
+  final Function(num) onButtonPressed;
+  final Color color;
+
+  const ApplyMethodItem({
+    Key? key,
+    required this.name,
+    required this.onButtonPressed,
+    required this.color,
+    this.defaultValue = 10,
+  }) : super(key: key);
+
+  @override
+  _ApplyMethodItemState createState() => _ApplyMethodItemState();
+}
+
+class _ApplyMethodItemState extends State<ApplyMethodItem> {
+  late num _value;
+
+  @override
+  void initState() {
+    _value = widget.defaultValue;
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Expanded(
+            child: Padding(
```

Comment:
not sure if your `flutter format` is working. broken like here

Reply:
It does work. So you mean I should add trailing commas where they are missing!?

Reply:
right now it is inconsistent across the file (some have 2 and some have 4 in the param passing).

look at the next line and you can tell this is off. are you sure everything is spaces? (maybe tab used here?)